solidstart-tanstackquery-tailwind-modules/src/components/UserProfile/index.tsx

```diff
@@ -9,103 +9,100 @@ import { Icon } from 'solid-heroicons';
 import { TwitterIcon } from '../Icons';
 import OrgList from './OrgList';
 import styles from './UserProfile.module.css';
-import { Show } from 'solid-js';
+import { Show, splitProps } from 'solid-js';
+import { UserProfileProps } from '~/types/user-profile-type';
```

Comment:
Why would you move the props for this component to a separate file?

I think the props type should be colocated  with the component.

Reply:
The props is been used in other places.